src/main.c

```diff
@@ -806,6 +809,12 @@ static int ubus_poe_info_cb(struct ubus_context *ctx, struct ubus_object *obj,
 			blobmsg_add_string(b, "status", state->ports[i].status);
 		else
 			blobmsg_add_string(b, "status", "unknown");
+		if (state->ports[i].mvolt)
+			blobmsg_add_double(b, "voltage", state->ports[i].mvolt / 1000.0);
+		if (state->ports[i].mampere)
+			blobmsg_add_double(b, "current", state->ports[i].mampere / 1000.0);
+		if (state->ports[i].tempc)
+			blobmsg_add_double(b, "temperature", state->ports[i].tempc);
```

Comment:
Only report the temperature if it's not 0 C? This doesn't look right

Reply:
got it, I'll see if I can map an absent value to nan.